src/components/Layout/index.tsx

```diff
@@ -63,13 +63,16 @@ export default function Layout(): JSX.Element | null {
             visible ? 'open left-0' : 'closed -left-full'
           }`}
         >
-          <img
-            className="pendulum-logo"
-            src={sideBarLogo}
-            alt=""
-            style={isPendulum ? {} : { marginTop: 20, marginBottom: 30, marginLeft: 30 }}
-          />
-          {isTestnet && <div className="foucoco-tag">Foucoco testnet</div>}
+          <div
+            style={
+              isPendulum ? { marginLeft: 20, marginBottom: 20 } : { marginTop: 20, marginBottom: 30, marginLeft: 30 }
+            }
+          >
+            <Link to={`/${network}/dashboard`} target="_blank">
```

Comment:
This needs to change to either https://pendulumchain.org/ when the network is Pendulum or https://pendulumchain.org/amplitude when the network is Foucoco/Amplitude.